client/state/happychat/constants.js

```diff
@@ -7,3 +7,9 @@ export const HAPPYCHAT_CONNECTION_ERROR_TRANSPORT_ERROR = 'transport error';
 
 // Max number of messages to save between refreshes
 export const HAPPYCHAT_MAX_STORED_MESSAGES = 30;
+
+// Groups
+export const HAPPYCHAT_GROUP_JPOP = 'jpop';
+export const HAPPYCHAT_GROUP_WOO = 'woo';
+export const HAPPYCHAT_GROUP_JPHP = 'jpphp';
```

Comment:
This is incredibly nitpicky, but should we make this `HAPPYCHAT_GROUP_JPPHP` so the constant name matches the commonly used acronym? I can see that tripping us up in the future?